src/classes/RD2_ElevateInformation_CTRL.cls

```diff
@@ -1,10 +1,8 @@
 /*
     Copyright (c) 2020, Salesforce.org
     All rights reserved.
-
```

Comment:
Is there a reason the blank lines were removed from all of these classes? The line space in the comment block does make it more readable.